File: quasar/Quasar.DataTransfer/Produtos/Requests/EspecificacaoInserirRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Quasar.DataTransfer.Produtos.Requests
{
    public class EspecificacaoInserirRequest
    {
        public string Posicao { get; set; }
        public string Cor { get; set; }
        public DateTime Ano { get; set; }
        public string Veiculo { get; set; }
    }
}

File: quasar-front/src/app/produtos/models/requests/especificacaoInserir.request.ts
export class EspecificacaoInserirRequest {
  public posicao: string;
  public cor: string;
  public ano: string;
  public veiculo: string;

  constructor(params: Partial<EspecificacaoInserirRequest>) {
    this.posicao = params.posicao || '';
    this.cor = params.cor || '';
    this.ano = params.ano || '';
    this.veiculo = params.veiculo || '';
  }
}
